misk-grpc-tests/build.gradle

```diff
@@ -28,9 +29,15 @@ protobuf {
   }
 }
 
+wire {
+  kotlin {
+  }
+}
+
 sourceSets {
   main.java.srcDirs += 'build/generated/source/proto/main/grpc'
   main.java.srcDirs += 'build/generated/source/proto/main/java'
+  main.java.srcDirs += 'build/generated/src/main/java'
```

Comment:
was this necessary?  the plugin should add this automatically.

Reply:
Removed.